kubernetes/spec/schema.json

```diff
@@ -0,0 +1,2246 @@
+{
+  "$id": "https://serverlessworkflow.io/schemas/1.0.0/workflow.yaml",
```

Comment:
This is a recurrent tech debt we have with other SDKs as well. Instead of reading from local, can we read from a URL? We have this: https://serverlessworkflow.io/schemas/1.0.0-alpha2/workflow.json

I'll automate publishing these schemas.